activerecord/test/models/clothing_item_inventory.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class ClothingItemInventory < ActiveRecord::Base
+  belongs_to :clothing_item, primary_key: [:color, :clothing_type],
+    foreign_key: [:clothing_item_color, :clothing_type]
```

Comment:
As this progresses using `clothing_type` and `color` for the query constraints and associations is making it hard to wrap my mind around how the items are associated.

I think we should change the examples to use a more realistic key grouping like `[:id, :blog_id]` or something like that where blog_id represents the "tenant". If you want to stick with clothes maybe we can do `[:id, :store_id]`.

Reply:
>something like that where blog_id represents the "tenant". 

For some reason I used blog, blog posts and comments models setup in the commit message as an example but end up using `ClothingItemInventory` in the code. I agree that inventory association is not the easiest to grasp so I'm going to switch to blog setup to be used in association tests.